test/e2e/instrumentation/monitoring/accelerator.go

```diff
@@ -60,9 +61,9 @@ func testStackdriverAcceleratorMonitoring(f *framework.Framework) {
 	projectID := framework.TestContext.CloudConfig.ProjectID
 
 	ctx := context.Background()
-	client, err := google.DefaultClient(ctx, gcm.CloudPlatformScope)
+	client, _ := google.DefaultClient(ctx, gcm.CloudPlatformScope)
```

Comment:
Should this error be handled？

Reply:
I updated "What this PR does / why we need it:" so please check it. this `err` is never used, so I removed it.

Reply:
you should handle this error instead of ignoring it. This is not a good habit for golang.

Reply:
there is a tool in golangci-lint, name is `errcheck`, it is work on find unhandled errors

Reply:
I'm not sure whether we should handle this error or not. 
/assign @SataQiu